<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>265</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>573</width>
    <height>265</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>573</width>
    <height>265</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&amp;Имя хоста</string>
     </property>
     <property name="buddy">
      <cstring>LE_Host</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>&amp;Имя БД</string>
     </property>
     <property name="buddy">
      <cstring>LE_DB</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>&amp;Логин</string>
     </property>
     <property name="buddy">
      <cstring>LE_Login</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>&amp;Пароль</string>
     </property>
     <property name="buddy">
      <cstring>LE_Pass</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>&amp;Порт подключения к БД</string>
     </property>
     <property name="buddy">
      <cstring>SB_Port</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="LE_Host">
     <property name="maxLength">
      <number>20</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="LE_DB"/>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="LE_Login"/>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="LE_Pass"/>
   </item>
   <item row="9" column="1">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Отправить</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSpinBox" name="SB_Port"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>LE_Host</tabstop>
  <tabstop>LE_DB</tabstop>
  <tabstop>LE_Login</tabstop>
  <tabstop>LE_Pass</tabstop>
  <tabstop>SB_Port</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
